lib/api/v3/utilities/path_helper.rb

```diff
@@ -46,6 +46,14 @@ def self.attachment(id)
             "#{root}/attachments/#{id}"
           end
 
+          def self.attachment_download(id)
+            "#{root_path}attachments/#{id}"
+          end
```

Comment:
RFC: I could also exclude this link form the APIv3 path helper and use the helpers of rails instead.

At this point I am unsure how to balance rails-independence vs. code repetition...